libraries/apollo-testing-support-internal/src/linuxMain/kotlin/com/apollographql/apollo/testing/currentThreadId.linux.kt

```diff
@@ -0,0 +1,5 @@
+package com.apollographql.apollo.testing
+
+actual fun currentThreadId(): String {
+  return "linux"
```

Comment:
```suggestion
  return pthread_self().toString()

```

Reply:
Right, JS is always single threaded so the thread id is always the same there but it's not the case on linux.